scheduler/src/main/java/gov/nasa/jpl/aerie/scheduler/constraints/TimeRangeExpression.java

```diff
@@ -205,14 +192,13 @@ public Builder name(String name) {
     /**
      * relative instant use case
      *
-     * @param state x
+     * @param nameDiscreteProfile name of the discrete profile
```

Comment:
[minor] same comment about `nameThing` vs `thingName` (I wrote these comments in reverse order, see below)